tests/test-backend-ops.cpp

```diff
@@ -6480,6 +6515,15 @@ static std::vector<std::unique_ptr<test_case>> make_test_cases_eval() {
         }
     }
 
+    // single inplace test per type/mode/ff
+    for (ggml_type type : {GGML_TYPE_F32, GGML_TYPE_F16}) {
+        for (int mode : {0, 2, 8, 24}) {
```

Comment:
It's better to start using the `GGML_ROPE_TYPE_` macros instead of hardcoded constants for the `mode`. Can also update the few other places where we use int constants for this.

@slaren Should we add:

```c
#define GGML_ROPE_TYPE_NORM   0
```

It's currently missing and I think it will improve consistency if we start using it.